src/git/url_prefix.js

```diff
@@ -57,7 +57,19 @@ function getGithubURLPrefix(root) {
       sha = head;
     }
     if (sha) {
-      const parsed = gitUrlParse(getRemoteOrigin.sync(root));
+      let origin;
+      if (git.indexOf(root) === 0) {
+        const config = ini.parse(
```

Comment:
For documentation.js's purposes this probably isn't important, but git configuration is a little different than ini and the underlying parse-git-config module [did a find/replace](https://github.com/jonschlinkert/parse-git-config/blob/master/index.js#L150) to preprocess it so it would work as an ini.

Reply:
Oh, good point! I did have `parse-git-config` here first, but then I upgraded its version to the latest and everything broke, and I didn't have enough patience to get to the bottom of it, so wrote without a dep instead. Will make sure it doesn't break when submodule name contains dots, and if it is, I'll submit another PR. Thanks!